script/RumpelConfig.sol

```diff
@@ -638,6 +669,239 @@ library RumpelConfig {
         return configs;
     }
 
+    // Mellow Vaults ----
+    function getMellowVaultsGuardProtocolConfigs() internal pure returns (ProtocolGuardConfig[] memory) {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](17);
+
+        // Mellow Simple Lido DV
+        configs[0] = ProtocolGuardConfig({target: MAINNET_MELLOW_DVSTETH, selectorStates: new SelectorState[](2)});
+        configs[0].selectorStates[0] =
+            SelectorState({selector: IMellow.deposit.selector, state: RumpelGuard.AllowListState.ON});
```

Comment:
Looks like the IMellow.deposit function isn't the same for all vaults. I checked `MAINNET_MELLOW_DVSTETH` and `MAINNET_MELLOW_RENZO_PZETH ` and while they both have deposit functions, the parameters are different from the current IMellow.deposit function and different than each other - causing different function selectors.

Probably need to review each vault's deposit and registerWithdrawal functions and either confirm it matches the current IMellow interface and leave as is, or if it doesn't match, add a new interface with the different function below.